conbench/tests/app/test_runs.py

```diff
@@ -119,41 +118,7 @@ def test_baseline_run_links(self, client):
         self._assert_baseline_link(
             response.text,
             "latest_default",
-            benchmark_results[15].run_id,  # the latest default branch run in general
+            # earliest benchmark result on the latest default-branch commit
+            benchmark_results[9].run_id,
```

Comment:
This and similar changes to tests are because we're now tiebreaking the contender baseline runs by earliest benchmark result timestamp instead of run timestamp (which no longer exists). It still makes sense conceptually.